src/wrap-store/wrapStore.js

```diff
@@ -99,7 +99,9 @@ export default (store, {
   /**
    * Setup external action handler
    */
-  chrome.runtime.onMessageExternal.addListener(dispatchResponse);
+  if (chrome.runtime.onMessageExternal) {
```

Comment:
@vhmth Off the top of my head, no, I don't know of any existing library which does or aims to do that. I like the idea of a separate library which would be general enough to fit all extensions, rather than be specific to react-chrome-redux... Fits the unix philosophy of a utility doing one thing well.